### PR TITLE
feat: improve copy

### PR DESCRIPTION
**Pull request by dergigi**
Went over the main pages:

- Receive
- Jam
- Earn
- Send

### Screenshots 📸

01-receive
02-jam
02b-jam
03a-earn
03b-earn
03c-earn
04-send

### CODE REVIEW

File: src/components/Earn.jsx [hunk 1]
```diff
@@ -258,6 +258,11 @@ export default function Earn() {
 
           {serviceInfoAlert && <rb.Alert variant={serviceInfoAlert.variant}>{serviceInfoAlert.message}</rb.Alert>}
 
+          {!serviceInfo?.coinjoinInProgress &&
+            !serviceInfo?.makerRunning &&
+            !isWaitingMakerStart &&
+            !isWaitingMakerStop && <p className="text-secondary">{t('earn.market_explainer')}</p>}
```
Comment (dnlggr) [inline src/components/Earn.jsx:264]:
```suggestion
            !isWaitingMakerStop && <p className="text-secondary mb-4">{t('earn.market_explainer')}</p>}
```

File: src/i18n/locales/en/translation.json [hunk 2]
```diff
@@ -151,11 +151,11 @@
   },
   "send": {
     "loading": "Loading",
-    "title": "Send bitcoin",
-    "subtitle": "Collaborative transactions increase the privacy of yourself and others.",
+    "title": "Send",
+    "subtitle": "Use a collaborative transaction for increased security and privacy.",
```
Comment (dnlggr) [inline src/i18n/locales/en/translation.json:155]: What's the security aspect to it?

**Review by dnlggr — APPROVED** [review]
Awesome! 👏 ✅ 💬

File: src/i18n/locales/en/translation.json (hunk 2 above)
Comment (dergigi) [reply src/i18n/locales/en/translation.json:155]: Two sides of the same coin, see [Lopp's List](https://github.com/jlopp/physical-bitcoin-attacks).

There is a reason why the *s* in `https` stands for "secure" even though it is technically "private" - I think we should slowly but surely get this into the heads of bitcoiners too.

Comment (dnlggr) [reply src/i18n/locales/en/translation.json:155]: Fair point! 👍

**Comment by dergigi** [comment]
Added screenshots for easier review. Please have another look @dnlggr - nothing set in stone of course, but I tried to be as careful and consistent as possible with the copy.

I think pointing out that Jam allows you to "improve the privacy of yourself and others" is important. I also was careful to not mention "bitcoin" too often, since it is always confusing (the network? the currency? UTXOs? what?).

If you don't like the "security" mention we can remove it of course, but it is only mentioned once. I hope that we will be able to improve the wording and terminology over time everywhere, but I feel like this is already a big step in the right direction.

File: src/i18n/locales/en/translation.json [hunk 3]
```diff
@@ -202,22 +203,23 @@
     "text_getting_address": "Getting new address"
   },
   "earn": {
-    "title": "Earn bitcoin",
-    "subtitle": "By making your bitcoin available for others, you help them improve their privacy and can also earn a yield.",
+    "title": "Earn",
+    "subtitle": "By providing liquidity to the market, you improve the privacy of yourself and others. Your sats always remain under your control and never leave your wallet.",
+    "market_explainer": "You can choose how much you want to charge for your offer. Note that you participate and compete in an open market—there is no guarantee that other market participants will take you up on your offer. Being chosen for a collaborative transaction can take some time. Low fees increase your chances.",
```
Comment (dnlggr) [inline src/i18n/locales/en/translation.json:208]: > Low fees increase your chances.

Low fees and fidelity bonds, but we can't say that (yet). 😄

**Review by dnlggr — APPROVED** [review]
Looks good! ✅ 👍 

> If you don't like the "security" mention we can remove it of course,

After thinking about it again, I do get why "security" makes sense. Especially since we're trying to work towards improved terminology in the longer term.

> I think pointing out that Jam allows you to "improve the privacy of yourself and others" is important.

💯

**Pull request merged**